api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -1059,11 +1059,32 @@ public void testReaderUpdateWorkspaceThrows() {
     request.setWorkspace(ws);
     stubGetWorkspace(ws.getNamespace(), ws.getId(), ws.getCreator(), WorkspaceAccessLevel.READER);
     workspacesController.updateWorkspace(ws.getNamespace(), ws.getId(), request);
+  }
+
+  @Test(expected = ForbiddenException.class)
+  public void testWriterUpdateWorkspaceThrows() {
```

Comment:
split `testReaderUpdateWorkspaceThrows` in 2 because it was confusing.  Not related to the main purpose of this PR.